sct-app/src/test/java/org.lfenergy.compas.sct.app/SclAutomationServiceTest.java

```diff
@@ -40,6 +42,8 @@ void createSCD() throws Exception {
         assertEquals(1, expectedSCD.getCurrentElem().getIED().size());
         assertNotNull(expectedSCD.getCurrentElem().getDataTypeTemplates());
         assertEquals(2, expectedSCD.getCurrentElem().getCommunication().getSubNetwork().size());
+        MarshallerWrapper marshallerWrapper = createWrapper();
```

Comment:
Couldn't it be added to other tests?

Reply:
- added to other tests in `SclAutomationServiceTest`, `IEDAdapterTest`, `SclServiceTest` and `SclRootAdapterTest`.
- Also, created bug #128